src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileServiceImpl.java

```diff
@@ -64,6 +91,24 @@ public ConcurRequestExtractFileValidationService getConcurRequestExtractFileVali
         return this.concurRequestExtractFileValidationService;
     }
 
+    private String parseRequestExtractFileNameFrom(String requestExtractFullyQualifiedFileName) {
+        //plus 1 because we do not want the forward slash
+        return requestExtractFullyQualifiedFileName.substring((requestExtractFullyQualifiedFileName.lastIndexOf(ConcurConstants.FORWARD_SLASH)) + 1);
+    }
+
+    private ConcurRequestExtractFile loadFileIntoParsedDataObject(String requestExtractFullyQualifiedFileName) {
+        Object parsedFile = loadFile(requestExtractFullyQualifiedFileName);
+        List<ConcurRequestExtractFile> requestExtractFiles = (ArrayList<ConcurRequestExtractFile>) parsedFile;
+        if (requestExtractFiles.isEmpty() || requestExtractFiles.size() != 1) {
```

Comment:
why can't this IF statement just be the size check not equal 1, en empty array will return zero for the size, right?